firmware/robot/cpu/kicker.c

```diff
@@ -27,14 +27,7 @@ static int kicker_test_measure()
 void kicker_test()
 {
 	failures &= ~Fail_Kicker_Charge;
-	
-	if (base2008)
-	{
-		// Can't tell anything
-		kicker_charge = 1;
```

Comment:
@jjones646 and @ChrisDoho, this was related to the stuff you guys to get that robot to work without a kicker, right?